src/main/java/com/icthh/xm/ms/balance/service/mapper/PocketMapper.java

```diff
@@ -15,6 +15,7 @@
     @Mapping(source = "metadata.metadata", target = "metadata")
     PocketDTO toDto(Pocket pocket);
 
+    @Mapping(target = "subtractAmount", ignore = true)
```

Comment:
please explain why this new annotations is needed

Reply:
Same as above